docs/index.rst

```diff
@@ -762,6 +762,7 @@ Options:
     details.
 :orm_mode: allows usage of :ref:`ORM mode <orm_mode>`
 :alias_generator: callable that takes field name and returns alias for it
+:keep_untouched: custom types of model attributes (e. g. descriptors) that won't change during model creation
```

Comment:
```suggestion
:keep_untouched: set of types (e. g. descriptors) that won't change during model creation and won't be
  included in the model schemas.
```

Sorry, should have caught this the first time.

Reply:
NP. But how about change 'set of types' to 'tuple of types'? I guess you meant 'set'  in general, but I still find this a bit confusing. What do you think?

Reply:
yes of course, my mistake.